lang/src/org/partiql/lang/eval/BagOp.kt

```diff
@@ -0,0 +1,113 @@
+package org.partiql.lang.eval
+
+import com.amazon.ion.IonContainer
+import com.amazon.ion.IonStruct
+import com.amazon.ionelement.api.MetaContainer
+import org.partiql.lang.domains.PartiqlAst
+import org.partiql.lang.domains.PartiqlPhysical
+import org.partiql.lang.errors.ErrorCode
+import org.partiql.pig.runtime.DomainNode
+
+fun interface ExprValueBagOp {
```

Comment:
I'm not sure this needs a kdoc since it's not a public API, but I've included it. It can be an internal class, but the package structure of `eval` is non-existent. Once refactored, this will cease to exist or be moved depending on the expr abstraction. Kind of redundant and cluttering to define such obvious comments on abstractions, but the RFC link is worthwhile

Reply:
Fair point, thanks for explaining.